addons/account/models/account_journal_dashboard.py

```diff
@@ -117,7 +118,7 @@ def build_graph_data(date, amount):
             date = val['date']
             if date != today.strftime(DF):  # make sure the last point in the graph is today
                 data[:0] = [build_graph_data(date, amount)]
-            amount -= val['amount']
+            amount = currency.round(amount - val['amount'])
```

Comment:
Keep as it is but add the following line after the loop:

`amount = currency.round(amount)`